doc/tor.md

```diff
@@ -110,8 +110,8 @@ are seen with `-debug=tor` they can be resolved by adding both the user running
 the user running bitcoind to the same group and setting permissions appropriately. On
 Debian-based systems the user running bitcoind can be added to the debian-tor group,
 which has the appropriate permissions. An alternative authentication method is the use
-of the `-torpassword` flag and a `hash-password` which can be enabled and specified in
-Tor configuration.
+of the `-torpassword=hashpassword` option. The `hashpassword` can be enabled and specified in
+Tor configuration under the `HashedControlPassword` option.
```

Comment:
it's an improvement but still not fully clear as to what actually needs to be done to set up tor password authentication. to be clear:
- bitcoind's `torpassword` takes the *unhashed* password
- tor's configuration `HashedControlPassword` has a hashed password

the way that this is worded, it sounds like bitcoind's argument takes the hashes password too

Reply:
I just pushed an other change that adds more details on the relation between `password` and `HashedControlPassword`.